docs/antora.yml

```diff
@@ -23,6 +23,6 @@ nav:
 ext:
   collector:
     run:
-      command: make reference-docs
+      command: make all-docs
     scan:
-      dir: dist/reference
+      dir: dist
```

Comment:
Will the other files in this directory, e.g. `ec_linux_amd64`, cause any issues?

Reply:
Best keep this separate, e.g. if we need to delete all the generated `.adoc` files we can remove just `dist/reference` and not care if there are any other files in `dist`. Might be better to have two collectors here.

Reply:
@zregvart, this isn't for laying down files, it's for scanning. 

To @lcarva's question, no it won't care about other files in the directory.